frontend/src/components/TopicSidebarShell.css

```
@@ -1274,6 +1274,59 @@
+
+.topic-row__dm-glyph {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  line-height: 1;
+}
+
+.topic-row__badge--dm .topic-row__dm-glyph svg {
+  width: 12px;
+  height: 12px;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)